docs/contributor-guide.md

```diff
@@ -62,23 +62,29 @@ See the `tests/integration/test_container.py` for examples.
 
 ## Talk to us
 
-Use Github [discussions] forum or for a live chat experience try
-`#ansible-devtools` IRC channel on libera.chat or Matrix room
-[#devtools:ansible.com](https://matrix.to/#/#devtools:ansible.com).
+- Join the Ansible forum:
 
-For the full list of Ansible IRC and Mailing list, please see the [Ansible
-Communication] page. Release announcements will be made to the [Ansible
-Announce] list.
+  - [Get Help](https://forum.ansible.com/c/help/6): get help or help others. Please add appropriate tags if you start new discussions, for example the `devtools` tag.
+  - [Posts tagged with 'devtools'](https://forum.ansible.com/tag/devtools): subscribe to participate in project-related conversations.
+  - [Social Spaces](https://forum.ansible.com/c/chat/4): gather and interact with fellow enthusiasts.
+  - [News & Announcements](https://forum.ansible.com/c/news/5): track project-wide announcements including social events.
+
+- The Ansible [Bullhorn newsletter]: used to announce releases and important changes.
```

Comment:
@Andersson007 is there a missing link to the bullhorn?

Reply:
Oh nvm! Sorry I missed the reference at the end and forgot that you could do that sort of linking with MD. Was expecting a url in parentheses.

Reply:
I don't personally like this style, I'll change it, looks confusing to me too